### PR TITLE
Recipe Page is now styled

### PR DESCRIPTION
**Pull request by ez-may**
Picture for reference:
image

### CODE REVIEW

File: src/main/webapp/css/recipe-page.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#image-description {
+  align-items: flex-start;
+  display: flex;
+  left: 10px;
+  position:relative;
+}
+
+#description {
+  font-size: 20px;
+  margin: 0px 15px 5px 10px;
```
Comment (oturner24) [inline src/main/webapp/css/recipe-page.css]: super nit: you don't need units on 0. 0px -> 0

File: src/main/webapp/css/recipe-page.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  background-color: #ffffff;
+  border-color: #195e83ab;
+  border-radius: 10px;
+  border-style: solid;
+}
+
+#recipe-name {
+  font-size: 40px;
+  position: relative;
+  left: 10px;
```
Comment (oturner24) [inline src/main/webapp/css/recipe-page.css]: nit: think these need to be swapped to be alphabetized

**Review by oturner24 — APPROVED** [review]

File: src/main/webapp/javascript/recipe-rendering.js [hunk 3]
```diff
@@ -76,6 +76,7 @@ createRecipeCard = (divID, recipeInfo) => {
  */
 function redirectRecipePage(id) {
   window.location.assign("/pages/RecipePageTemplate.jsp?id=" + id.toString());
+
```
Comment (isamitre) [inline src/main/webapp/javascript/recipe-rendering.js]: nit: no need for extra line here

**Review by isamitre — APPROVED** [review]
LGTM

File: src/main/webapp/css/recipe-page.css (hunk 2 above)
Comment (ez-may) [reply src/main/webapp/css/recipe-page.css]: Ack and fixed

File: src/main/webapp/css/recipe-page.css (hunk 1 above)
Comment (ez-may) [reply src/main/webapp/css/recipe-page.css]: Ack and fixed

File: src/main/webapp/javascript/recipe-rendering.js (hunk 3 above)
Comment (ez-may) [reply src/main/webapp/javascript/recipe-rendering.js]: Ack and fixed

**Pull request merged**